### ISSUE TITLE
Get chunks of data in real time

### ISSUE THREAD

**Issue by cogmeta** [hackingbeauty/react-mic#27]
Is it possible to pass a function to ReactMic compoment and get the audio data chunks as an when they become available so that the data can be sent to server for real time processing?

**Comment by hackingbeauty** [comment]
Yes I'm happy to merge any additional functionality that works and makes sense.

I don't have the bandwidth to add your functionality at this time.  Do you want to submit a PR?  :)

**Comment by cogmeta** [comment]
Thanks, Mark. We will submit the changes to the code soon.

**Comment by daas-ankur-shukla** [comment]
@cogmeta any development on the functionality you mentioned? I would also like to have it.
